packages/pluginutils/rollup.config.js

```diff
@@ -13,14 +15,7 @@ export default {
   ],
   external: Object.keys(pkg.dependencies).concat('path', 'util'),
   output: [
-    {
-      format: 'cjs',
-      file: pkg.main,
-      exports: 'named'
-    },
-    {
-      format: 'es',
-      file: pkg.module
-    }
+    { format: 'cjs', file: pkg.main, exports: 'named' },
+    { file: pkg.module, format: 'es', plugins: [emitModulePackageFile()] }
```

Comment:
```suggestion
    { format: 'cjs', file: pkg.main, exports: 'named' },
    { format: 'es', file: pkg.module, plugins: [emitModulePackageFile()] }
```

To have both outputs the same format order.

Reply:
```suggestion
    { file: pkg.main,  format: 'cjs', exports: 'named' },
    { file: pkg.module, format: 'es', plugins: [emitModulePackageFile()] }
```

If you prefer the other order.

Reply:
Also I recommend the same in this PR: https://github.com/rollup/plugins/pull/413